database_cleanup/models/purge_fields.py

```diff
@@ -0,0 +1,123 @@
+# Copyright 2014-2016 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+# pylint: disable=consider-merging-classes-inherited
+from odoo import _, api, models, fields
+from odoo.exceptions import UserError
+from odoo.addons.base.models.ir_model import MODULE_UNINSTALL_FLAG
+from ..identifier_adapter import IdentifierAdapter
+
+
+class CleanupPurgeLineField(models.TransientModel):
+    _inherit = 'cleanup.purge.line'
+    _name = 'cleanup.purge.line.field'
```

Comment:
How about adding a test [here](https://github.com/OCA/server-tools/blob/74bee5ff8678ef01db56e6d93aa5734f82f8d9b1/database_cleanup/tests/test_database_cleanup.py#L71) that, after creation of the dummy model, creates a field on it as well, then purges the field?

A test would also make it easier to review when porting this feature to versions 13.0 and higher.

Reply:
Or if you like, throw in a review [here](https://github.com/OCA/server-tools/pull/2295) so that I can merge it, and the tests will be that much nicer.